src/api/hooks/News.tsx

```diff
@@ -8,8 +8,15 @@ export const useNewsById = (id: number) => {
   return useQuery<News, RequestResponse>([QUERY_KEY, id], () => API.getNewsItem(id), { enabled: id !== -1 });
 };
 
-export const useNews = () => {
-  return useInfiniteQuery<PaginationResponse<News>, RequestResponse>(QUERY_KEY, () => API.getNewsItems());
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const useNews = (filters?: any) => {
+  return useInfiniteQuery<PaginationResponse<News>, RequestResponse>(
+    [QUERY_KEY, filters],
+    ({ pageParam = 1 }) => API.getNewsItems({ ...filters, page: pageParam }),
+    {
+      getNextPageParam: (lastPage) => lastPage.next,
+    },
+  );
```

Comment:
Du kan fjerne filters siden det ikke brukes i News
```suggestion
export const useNews = () => {
  return useInfiniteQuery<PaginationResponse<News>, RequestResponse>(
    [QUERY_KEY, filters],
    ({ pageParam = 1 }) => API.getNewsItems({ page: pageParam }),
    {
      getNextPageParam: (lastPage) => lastPage.next,
    },
  );
```